syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -448,89 +457,34 @@ def create_pointer(
     ) -> PointerTensor:
         """Creates a pointer to the "self" torch.Tensor object.
 
-        This method is called on a torch.Tensor object, returning a pointer
```

Comment:
should the above definition be here with the docs removed?

Reply:
What do you mean exactly?

Reply:
The change here deletes much of the documentation but leaves the method

Reply:
Yes, because basically this method doesn't do anything anymore, so I didn't wanted to duplicate the docstring, as we now how hard it is to already keep one version up to date ;)